nova/virt/vmwareapi/host.py

```diff
@@ -124,9 +124,9 @@ def update_status(self):
             "numa_topology": None,
         }
 
-        for host, stats in per_host_stats.items():
-            data[host] = self._merge_stats(host, stats, defaults)
-
+        for host_ref, host in per_host_stats.items():
+            data[host["name"]] = self._merge_stats(host["name"], host,
+                                                   defaults)
```

Comment:
I'm not sure why you renamed `stats` to `host` here - are these not the stats? Would `info` be a better name? It's not really clear that `host` is a dict containing the stats and some info :)